### ISSUE TITLE
Wrong generation of generics

### ISSUE THREAD

**Issue by fpotera** [stoicflame/enunciate#581]
Hello.
We have the following code:

```java
public class MapEntry<K,V>
{
	private K key;
	private V value;

    public MapEntry() {}

	public MapEntry(K key, V value)
	{
		this.key = key;
		this.value = value;
	}

	public K getKey() { return key; }

    public void setKey(K key) {

        this.key = key;
    }

	public V getValue()
	{
		return value;
	}
    public void setValue(V value) {
        this.value = value;
    }
}
```

Enunciate generate swaager.json file:

```json
    "json_MapEntry" : {
      "type" : "object",
      "title" : "MapEntry",
          "properties" : {
            "key" : {
"description" : "",
"type" : "file"
            },
            "value" : {
"description" : "",
"type" : "file"
            }
          },
      "description" : ""
    }
```

"key" and "value" are wrong generated as type "file" instead of "object".

**Comment by stoicflame** [comment]
Fixed in 2.9

**Comment by stoicflame** [comment]
[Enunciate 2.9.0 has been released.](https://github.com/stoicflame/enunciate/releases/tag/v2.9.0)
